packages/yourloops/lib/team/team.hook.tsx

```diff
@@ -54,6 +54,7 @@ import SpinningLoader from '../../components/loaders/spinning-loader'
 const ReactTeamContext = createContext<TeamContext>({} as TeamContext)
 
 export const PRIVATE_TEAM_ID = 'private'
+export const PRIVATE_TEAM_NAME = 'private'
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏There is also a `const PRIVATE_TEAM_NAME = 'private'` defined in `header.ts`, it would be great to reuse the value defined here